src/schedule/schedule.11tydata.js

```diff
@@ -1,45 +1,65 @@
-const { google, outlook, office365, yahoo, ics } = require('calendar-link');
-const simpleIcons = require('simple-icons');
+const { google, outlook, office365, ics } = require('calendar-link');
 const { isBefore } = require('date-fns');
+const url = 'https://events.lunch.dev/discord';
+const location = 'Lunch Dev Community Discord at events.lunch.dev';
 
 module.exports = {
 	eleventyComputed: {
-		addToCalendarList: ({ title, description, date: start }) => {
+		googleLink: ({ title, description, date: start }) => {
 			if (isBefore(new Date(start), new Date())) {
 				return;
 			}
 
-			let event = { title, description, start, duration: [1, 'hour'] };
+			return google({
+				title,
+				description,
+				start,
+				duration: [1, 'hour'],
+				location,
+				url,
```

Comment:
Unfortunately, I'm not sure Google Calendar supports `url` — it might just be Outlook. What would you say about doing a bit of a switcheroo for Google Calendars, where we use the URL for the location?

```
location: url
```

Reply:
interesting. do you think this is something that might need to be considered at the calendar-link library level?

i had assumed that they would work because we use the url field for all of our meetings (Google org).

Reply:
The calendar-link library mentions the following statement on their page:

> The url field defaults to document.URL if a global document object exists. For server-side rendering, you should supply the url manually.

> Not all calendars support the guests and url fields.

I wonder if we could do a zoom type thing and add this info into the description?